torch/csrc/jit/passes/python_print.h

```diff
@@ -1,13 +1,18 @@
 #pragma once
 #include "torch/csrc/WindowsTorchApiMacro.h"
 #include <iostream>
+#include <vector>
+
+namespace at {
+  struct Tensor;
+}
 
 namespace c10 {
   struct Symbol;
 }
 
 namespace torch { namespace jit {
 struct Graph;
-TORCH_API std::ostream& PythonPrint(std::ostream& out, const Graph& graph);
+TORCH_API std::vector<at::Tensor> PythonPrint(std::ostream& out, const Graph& graph, bool enforce_importable=false);
```

Comment:
This looks a little bit weird on the PythonPrint interface, As we are doing import/export, the constant table is not getting serialized/deserialized, so basically we might lose some information on importing if no constant table is preserved (emitModule hook are storing them in memory, not in disk).

Reply:
import_method just handles importing the code. There will be another piece of code that handles encoding the constant table and restoring it. These hooks test the import mechanism for code separately from the end-to-end module tests which do not exist yet.